source/Quickstart-GKE.rst

```diff
@@ -0,0 +1,326 @@
+
+::
+
+   gcloud beta container --project <project name> clusters create "standard-cluster-1" --zone "us-central1-a" --username "admin" --cluster-version "1.11.7-gke.12" --machine-type "n1-standard-4" --image-type "COS" --disk-type "pd-standard" --disk-size "100" --scopes "https://www.googleapis.com/auth/devstorage.read_only","https://www.googleapis.com/auth/logging.write","https://www.googleapis.com/auth/monitoring","https://www.googleapis.com/auth/servicecontrol","https://www.googleapis.com/auth/service.management.readonly","https://www.googleapis.com/auth/trace.append" --num-nodes "3" --enable-cloud-logging --enable-cloud-monitoring --no-enable-ip-alias --network "projects/<project name>/global/networks/default" --subnetwork "projects/<project name/regions/us-central1/subnetworks/default" --addons HorizontalPodAutoscaling,HttpLoadBalancing --enable-autoupgrade --enable-autorepair
+
+You may wait a few minutes for the cluster to be generated.
+
+In the Google Cloud console, select your cluster and then select
+``Connect``. The connect statement configures command line access. After
+you have edited the statement, you may run the command in your local
+shell or in the Cloud Shell.
+
+For this document, we are using the local shell.
+
+::
+
+   gcloud container clusters get-credentials standard-cluster-1 --zone us-central1-a --project <project name>
+
+Installing the Operator
+-----------------------
+
+1. Use the ``git clone`` command to download the correct branch of the
+   percona-xtradb-cluster-operator repository.
+   **Note:** *In the git statement, you must specify the correct branch
+   with the -b option.*
+
+.. code:: bash
+
+       git clone -b release-0.3.0 https://github.com/percona/percona-xtradb-cluster-operator
+
+After the repository is downloaded, change the directory to run the rest
+of the commands in this document:
+
+::
+
+   cd percona-xtradb-cluster-operator
+
+2. Extensions of the Kubernetes API are custom resources. The Custom
+   Resource Definition for PXC adds the operator resources. This step is
+   only run once. After the Kubernetes API has been extended any
+   additional operator deployments use the same resources.
+
+.. code:: bash
+
+   kubectl apply -f deploy/crd.yaml
+
+The following confirmation is returned:
+
+::
+
+         customresourcedefinition.apiextensions.k8s.io/perconaxtradbclusters.pxc.percona.com created
+         customresourcedefinition.apiextensions.k8s.io/perconaxtradbbackups.pxc.percona.com created
+
+3. Create a namespace. The resource names must be unique within the
+   namespace and provide a way to divide cluster resources between users
+   spread across multiple projects.
+   You must replace the ``<namespace name>`` placeholder with a
+   descriptive name.
+
+.. code:: bash
+
+   kubectl create namespace <namespace name>
+
+The following confirmation is returned:
+
+::
+
+       namespace/<namespace name> created
+
+4. Set the context for the namespace, which saves the namespace for
```

Comment:
I think "create a new namespace and switch into it" is one step with two commands

Reply:
I don't see that sentence.

Reply:
I am about joining step 3 and step 4 into one step

Reply:
Done.

Reply:
where it is done?